### DISCUSSION TITLE
Deepset/haystack

### DISCUSSION THREAD

**Discussion by jlonge4** [Arize-ai/phoenix#2532] [General]
Can I use phoenix with my haystack/deepset pipeline? Thanks

**Comment by mikeldking** [comment]
@jlonge4 You can definitely instrument your LLM calls using any of the existing instrumentations https://github.com/Arize-ai/openinference

Unfortunately we have not instrumented Haystack quite yet but we definitely plan on expanding our instrumentation coverage. Would love to learn more about your use-case!

**Reply by jlonge4** [reply]
For sure! Perhaps the existing bedrock instrumentation could work as I am using the bedrock integration in haystack for both embedding and LLM inference 🤔
